src/loader_transaction.rs

```diff
@@ -0,0 +1,53 @@
+//! The `dynamic_transaction` module provides functionality for loading and calling a program
+
+use bincode::serialize;
+use solana_program_interface::loader_instruction::LoaderInstruction;
+// use dynamic_program::DynamicProgram;
```

Comment:
nit: remove

Reply:
Yeah, will be cleaned up before merge, the purpose of this initial preview is to make sure we are on the same page